tests/dune

```diff
@@ -1,4 +1,29 @@
+(library
+ (name test_server)
+ (modules test_server)
+ (libraries ssl threads))
+
 (test
  (name ssl_test)
- (libraries ssl alcotest)
- (deps digicert_certificate.pem))
```

Comment:
I'm not sure why you removed `deps` but tests are failing because the `.pem` and `.key` files aren't being copied to the build folder.